repo2docker/buildpacks/conda/install-base-env.bash

```diff
@@ -36,7 +36,11 @@ update_dependencies: false
 # channel_priority: flexible
 EOT
 
-micromamba install conda=${CONDA_VERSION} mamba=${MAMBA_VERSION} -y
+micromamba install \
+    -c conda-forge \
+    -c conda-forge/label/mamba_dev \
```

Comment:
Due to [this choice](https://github.com/conda-forge/mamba-feedstock/blob/652513b5ae38767a244953f367d7b3cfbfd6044b/recipe/conda_build_config.yaml#L2C17-L2C33):
```suggestion
    -c conda-forge/label/mamba_prerelease \
```